superset-frontend/src/components/Switch/Switch.test.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { render, screen } from 'spec/helpers/testing-library';
+import { Switch } from '.';
+
+const mockedProps = {
+  label: 'testLabel',
+  dataTest: 'dataTest',
+  checked: false,
+};
+
+test('should render', () => {
```

Comment:
I believe these components, in their most basic form, do not require these types of tests since Ant Design should already cover them. However, if we end up customizing them significantly, it would make sense to include these tests. For now, I think we can leave them here for reference.

Reply:
I think it's good to have even if just as a placeholder to enhance. It's a good assumption to know that `superset-frontend/src/components/*/*.tsx` has a matching `*.test.tsx`